test/vswhere.test/vswhere.test.vcxproj

```diff
@@ -71,6 +71,7 @@
       </AdditionalIncludeDirectories>
       <PreprocessorDefinitions>WIN32;_DEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <UseFullPaths>true</UseFullPaths>
+      <AdditionalOptions>/DYNAMICBASE %(AdditionalOptions)</AdditionalOptions>
```

Comment:
Seems unnecessary for a test project. Was this required?

Reply:
The 1ES template scans all artifacts published and runs binskim and other compliance tools against the contents. The test dll is in the artifact, so it was getting scanned and flagged.

We have 3 options: get an exception for the failure (should be easy since it is a test assembly), add the necessary flags, or remove the assembly from the artifact. I chose to just add the necessary flags for compliance to simplify. Since most flags are in the root cpp props, they come for free. 

The only thing I don't know for sure is whether /DYNAMICBASE is needed. The binskim docs state:
> Error: Error
'{0}' does not enable the control flow guard (CFG) mitigation. To resolve this issue, pass /guard:cf on both the compiler and linker command lines. Binaries also require the /DYNAMICBASE linker option in order to enable CFG.

I followed that guidance, but didn't test without the option, so I am unsure if it is truly needed or not.

Reply:
No problem. It was more of a question out of surprise. I'm surprised they don't make distinctions between shipping and non-shipping code, but that might fit their rather overarching pipeline classification, too.